libcloud/storage/drivers/backblaze_b2.py

```diff
@@ -292,7 +316,7 @@ def download_object_as_stream(self, obj, chunk_size=None):
                                 success_status_code=httplib.OK)
 
     def upload_object(self, file_path, container, object_name, extra=None,
-                      verify_hash=True, headers=None):
+                      verify_hash=True, headers=None, iterator=None):
```

Comment:
I just tried to implement the large file upload and it is not yet supported by backblaze. I'm going to go forward with this temporary ```upload_object_via_stream``` method making the changes you suggested.